src/dymaptic.GeoBlazor.Core/Components/Renderers/IImageryRenderer.cs

```diff
@@ -0,0 +1,19 @@
+﻿using dymaptic.GeoBlazor.Core.Components.Layers;
+
+namespace dymaptic.GeoBlazor.Core.Components.Renderers;
+
+/// <summary>
+///     This interface is to bridge . 
+///     <a target="_blank" href="https://developers.arcgis.com/javascript/latest/api-reference/esri-layers-ImageryLayer.html">
```

Comment:
finish comment on line 6, update (or remove) link so it points to the renderer docs

Reply:
corrected the href to point at the renderer section